eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java

```diff
@@ -135,6 +135,32 @@ public void stdinEmptyTest() {
         );
     }
 
+    @Test
+    public void stdinfewOneLineTest() {
+        final String first = "first-line";
```

Comment:
@includealex can you try to use Unicode texts here, instead of ASCII. Would be interesting to see how the test will behave.